### DISCUSSION TITLE
Ignoring whitespaces while comparing?

### DISCUSSION THREAD

**Discussion by svccsotest10** [romankh3/image-comparison#216] [Q&A]
I wanted to compare only text and styling changes between images. How can i ignore whitespaces while comparing?

**Background**:

I am comparing two webpages. Both webpages are of same source just deployed on two environments - env1 and env2.

Now env1 renders page with little whitespace of two lines from top while env2 renders without it. I have automated the process and of taking screenshot of webpages and then using image-comparison to compare the images from 2 env's

**Comment by romankh3** [comment]
hm...
It would be really hard to test, but it could be done.

You need to ignore all the places, which you don't want to compare.
